docs/migrations/00157.md

```diff
@@ -0,0 +1,25 @@
+#### Update API to use enhancer pattern
+
+This change updates the API from a plugin to an enhancer, and greatly simplifies the configuration.
+The following tokens are removed:
+
+- `CsrfExpireToken`
+- `FetchForCsrfToken`
+- `SessionToken`
+
+Instead of registering this plugin on the `FetchToken`, you can now use this as an enhancer on the `FetchToken`.
+
+```diff
+import CsrfProtection from 'fusion-plugin-csrf-protection';
+-app.register(SessionToken, Session);	  
+-app.register(FetchForCsrfToken, fetch);	  
+-app.register(FetchToken, CsrfProtection);	
+-if (__BROWSER__) {	
+-  app.register(FetchForCsrfToken, fetch);	
+-} 	
+-app.register(CsrfExpireToken, 60 * 60 * 24); 
++app.register(FetchToken, fetch);
++app.enhance(FetchToken, CsrfProtection)
```

Comment:
Is this accurate? Based on [this](https://github.com/fusionjs/fusion-plugin-csrf-protection/pull/157/files#diff-04c6e90faac2675aa89e2176d2eec7d8L68) you now import an enhancer which is distinct from the `CSRFProtection` plugin.

Reply:
The `CsrfProtection` variable is now an enhancer